gslib/commands/iam.py

```diff
@@ -137,8 +142,15 @@
   The gsutil -m option may be set to handle object-level operations more
   efficiently.
 
+  Note: The ``iam ch`` command may NOT be used to the change IAM policy of a
```

Comment:
Should we also clearly mention that not only they cannot update a policy with condition, they can not add condition to a policy with no condition neither.

Reply:
Sure, good call. I tweaked the wording a bit.